code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -71,6 +76,11 @@ obj/item/weapon/gun/energy/staff/special_check(var/mob/living/user)
 	fire_sound = 'sound/magic/wand.ogg'
 	max_shots = 10
 
+/obj/item/weapon/gun/energy/staff/animate/pickup(mob/living/user as mob)

```

Comment:
This is unnecessary because the parent proc already has this definition. Delete.